src/lib/bootloader/grub2.rb

```diff
@@ -79,12 +83,14 @@ def propose
       # boot, safer option for legacy booting (bnc#872054)
       self.pmbr_action = :add if Yast::BootStorage.gpt_boot_disk?
       device_map.propose if Yast::Arch.x86_64 || Yast::Arch.i386
+      @trusted_boot = true # FIXME: another default value
```

Comment:
I really do not think it is good idea to propose trusted boot to true by default